vllm/compilation/collective_fusion.py

```diff
@@ -1121,48 +1123,49 @@ def __init__(self, config: VllmConfig):
             # in fallback path, when we don't use flashinfer
             fuse_rms_quant=config.compilation_config.pass_config.enable_fusion)
 
-        for epsilon in [1e-5, 1e-6]:
-            AllReduceFusedRMSNormStaticQuantFP8Pattern(
-                epsilon,
-                self.model_dtype,
-                self.device,
-                self.allreduce_params,
-            ).register(self.patterns)
-            AllReduceFusedAddRMSNormStaticQuantFP8Pattern(
-                epsilon,
-                self.model_dtype,
-                self.device,
-                self.allreduce_params,
-            ).register(self.patterns)
-            if current_platform.has_device_capability(100):
-                AllReduceFusedRMSNormStaticQuantNVFP4Pattern(
+        with enable_fake_mode():
```

Comment:
In the interest of reducing the changes here:

```suggestion
        self.register_patterns()

    @enable_fake_mode()
    def register_patterns():
```